eng/common/pipelines/templates/archetype-typespec-emitter.yml

```diff
@@ -79,6 +84,8 @@ extends:
       - job: Build
         steps:
         - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+          parameters:
```

Comment:
@weshaggard @hallipr 
I realized these changes should be made in azure-sdk-tools repo instead.
Created https://github.com/Azure/azure-sdk-tools/pull/9373 for the change.